src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -44,10 +46,14 @@ def load_arguments(self, _):
         c.argument('admin_enabled', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
 
     with self.argument_context('acr repository delete') as c:
-        c.argument('manifest', nargs='?', required=False, const='', default=None, help='The sha256 based digest of manifest to delete')
+        c.argument('manifest', nargs='?', required=False, const='', default=None, help=argparse.SUPPRESS)
         c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
-        c.argument('repository', help='The name of the repository to delete.')
-        c.argument('tag', help='The name of tag to delete')
+        c.argument('repository', help="The name of the repository to delete. For example, 'hello-world'")
+        c.argument('tag', help=argparse.SUPPRESS)
+        c.argument('image', help="The name of the image to delete. For example, 'hello-world:latest' or 'hello-world@sha256:abc123'.")
```

Comment:
No need for `For example` here. If there is support for `long-help` in embedded arguments, that can be filled in with information about the format, or it can be part of the short description in a form like:

`The name of the image to delete, optionally containing an sha256 hash or tag.`